db/migrate/20190124154742_add_missing_not_null_constraints_to_source_ref.rb

```diff
@@ -0,0 +1,21 @@
+class AddMissingNotNullConstraintsToSourceRef < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null(:container_images, :source_ref, false)
```

Comment:
Don't worry about the parens just to appease rubocop.  We just need to add this method to the overrides.

Reply:
hm ok

Reply:
https://github.com/ManageIQ/guides/pull/335